### PR TITLE
size docs for bitcode

### PR DESCRIPTION
**Pull request by dnfield**
Helps give a canonical answer for why the release `Flutter.framework` is so big now, and additional details on measuring final app size for iOS.

/cc @timsneath @eseidelGoogle

### CODE REVIEW

File: src/docs/resources/faq.md [hunk 1]
```diff
@@ -371,6 +371,16 @@ binaries within the IPA, making the compression less efficient
 (see the [iOS App Store Specific Considerations][]
 section of Apple’s [QA1795][]).
 
+Since Flutter has started bundling bitcode, the size of the release
+Flutter.framework has vastly increased. This is because the release binary now
+contains a large blob of LLVM IR (bitcode). This is used by later process in
```
Comment (timsneath) [inline src/docs/resources/faq.md]: later *processes* (plural)?

File: src/docs/resources/faq.md [hunk 2]
```diff
@@ -371,6 +371,16 @@ binaries within the IPA, making the compression less efficient
...
+Xcode and Apple's App Store to produce a final binary with the latest compiler
+optimizations and features. The profile and debug frameworks contain only
```
Comment (timsneath) [inline src/docs/resources/faq.md]: contain only *a*

File: src/docs/resources/faq.md [hunk 3]
```diff
@@ -371,6 +371,16 @@ binaries within the IPA, making the compression less efficient
...
+"bitcode marker", and are more representative of the engine's actual binary
+size. Whether you ship with bitcode or not, the increased size of the release
+framework is stripped out during the final steps of the build. These steps
+happen after archiving your app and shipping it to the store.
+
 Of course, YMMV, and we recommend that you measure your own app.
```
Comment (timsneath) [inline src/docs/resources/faq.md]: With this extra text, we should probably edit this line. "YMMV" isn't great for a foreign reader anyway. 

Something like:

> To get results that are more representative, we recommend that you measure your own app.

File: src/docs/resources/faq.md (hunk 1 above)
Comment (sfshaza2) [reply src/docs/resources/faq.md]: Can we rephrase this a bit? Perhaps:

```
As of Flutter SDK 1.12, the release binary now includes LLVM IR (bitcode).
Xcode uses this bitcode to produce a final binary for the App Store 
containing the latest compiler optimizations and features.
The profile and debug frameworks contain only a _bitcode marker_,
and are more representative of the engine's actual binary size.
Whether you ship with bitcode or not, the increased size of the release
framework is stripped out during the final steps of the build.
These steps happen after archiving your app and shipping it to the store.
```

File: src/docs/resources/faq.md (hunk 3 above)
Comment (sfshaza2) [reply src/docs/resources/faq.md]: +1

File: src/docs/resources/faq.md [hunk 4]
```diff
@@ -371,6 +371,16 @@ binaries within the IPA, making the compression less efficient
 (see the [iOS App Store Specific Considerations][]
 section of Apple’s [QA1795][]).
 
+Since Flutter has started bundling bitcode, the size of the release
+Flutter.framework has vastly increased. This is because the release binary now
```
Comment (xster) [inline src/docs/resources/faq.md]: Talk about what this means for people who don't see .frameworks and see .app or .ipas via xcode or flutter build ios. i.e. is this an add-to-app specific note?

File: src/docs/resources/faq.md [hunk 5]
```diff
@@ -371,6 +371,16 @@ binaries within the IPA, making the compression less efficient
...
+contains a large blob of LLVM IR (bitcode). This is used by later process in
+Xcode and Apple's App Store to produce a final binary with the latest compiler
+optimizations and features. The profile and debug frameworks contain only
+"bitcode marker", and are more representative of the engine's actual binary
+size. Whether you ship with bitcode or not, the increased size of the release
+framework is stripped out during the final steps of the build. These steps
```
Comment (xster) [inline src/docs/resources/faq.md]: Be specific here. How would a user validate their relief of this concern? Is this from an xcode output? Is it from their app connect webpage?

File: src/docs/perf/app-size.md [hunk 6]
```diff
@@ -88,6 +88,12 @@ The resulting IPA file for the `example/helloworld` app
 -rw-r--r--  1 userName  primarygroup   8.3M Oct 25 13:47 build/app.ipa
 ```
 
+A closer result can be obtained by creating a release archive as described in
+the [iOS create build archive instructions][]. If bitcode is enabled on your
+project, you will also have the option to rebuild from bitcode. You can also
```
Comment (xster) [inline src/docs/perf/app-size.md]: Describe what 'rebuild from bitcode' means and whether the user should or should not do so.

File: src/docs/resources/faq.md (hunk 5 above)
Comment (dnfield) [reply src/docs/resources/faq.md]: That's in the next paragraph.

File: src/docs/resources/faq.md (hunk 3 above)
Comment (dnfield) [reply src/docs/resources/faq.md]: Done

File: src/docs/perf/app-size.md (hunk 6 above)
Comment (dnfield) [reply src/docs/perf/app-size.md]: Added a line.

**Comment by dnfield** [comment]
@xster let me know if you think this still needs more details anywhere.  I'd be happy to open a follow up to address.

**Pull request merged**